lib/extensions/healthchecker/src/index.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+const url = require('url');
+
+const routes = require('abacus-router')();
+const webapp = require('abacus-webapp');
+const oauth = require('abacus-oauth');
+const throttle = require('abacus-throttle');
+const breaker = require('abacus-breaker');
+const retry = require('abacus-retry');
+const request = breaker(retry(require('abacus-request')));
+const yieldable = require('abacus-yieldable');
+const appuri = require('abacus-dataflow').sink;
+const uriurl = require('abacus-urienv').url;
+const tmap = yieldable(require('abacus-transform').map);
+
+const debug = require('abacus-debug')('abacus-healthchecker');
+const edebug = require('abacus-debug')('e-abacus-healthchecker');
+
+const api = process.env.CF_API;
+const domain = process.env.CF_DOMAIN;
+const protocol = url.parse(api).protocol;
+
+const credentials = 'Basic ' + new Buffer(process.env.CLIENT_ID
+  + ':' + process.env.CLIENT_SECRET).toString('base64');
+
+const scopes = process.env.CLIENT_SCOPES;
+const secured = process.env.SECURED === 'true';
+
+const authHeader = () => secured ?
+  { headers: { Authorization: credentials } } : {};
+
+const toJson = (string) => {
```

Comment:
this function and applicationGroups seems to be tightly coupled. The error message would be inappropriate for other env.vars

Let's merge the functions to avoid possible misleading errors when reusing